css/index.css

```diff
@@ -0,0 +1,91 @@
+body {
+font-family: Arial, Helvetica, sans-serif;
+}
+
+form { 
+border: 3px solid #f1f1f1;
+}
+
+nav {
+background-color: indigo;
+height: 40px;
+color: ivory;
+padding: 10px;
+}
+
+.form-container {
+margin: 50px 70px 60px 350px;
+width: 500px;
+height: auto;
+padding: 10px;
+}
+
+p {
+text-align: center;
+}
+
+h1 {
+text-align: center;
+color: indigo;
+}
+
+input[type=text], input[type=password] {
+  width: 100%;
+  padding: 10px 10px;
+  margin: 0 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  box-sizing: border-box;
+  border-radius: 10px;
+}
+
+input[type=checkbox] {
+margin-left: 130px;
+}
+
+.psw { 
+margin-top: -20px;
+margin-left: 112px;
+}
+
+.form-group {
+  margin-left: 95px;
+  padding: 10px;
+
+}
+
+
+button[type="submit"] {
```

Comment:
@indalgit use classname of login submit button for styling, do same for every single element/tag.